wgpu/backends/rs.py

```diff
@@ -176,6 +176,20 @@ def check_struct(struct_name, d):
 libf = SafeLibCalls(lib, error_handler)
 
 
+def enumerate_adapters():
+    """Return a list of all available adapters."""
+    # The first call is to get the number of adapters, and
+    # the second call is to get the actual adapters
```

Comment:
Let's mention the option here.
```suggestion
    # The first call is to get the number of adapters, and
    # the second call is to get the actual adapters.
    # Note that the second arg (now NULL) can be a
    # `WGPUInstanceEnumerateAdapterOptions` to filter by backend.
```